server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -122,9 +137,18 @@
     @Size(max = 255)
     private String arches;
 
+    @XmlTransient
+    @Column(name = "entity_version")
+    private Integer entityVersion;
+
+    @XmlTransient
+    @Column
+    private Integer locked;
```

Comment:
Did you try using `@Type(type = "org.hibernate.type.NumericBooleanType")`?  I don't have a problem with it being an integer in the database, but we can instruct it in how to convert the integers back and forth to booleans.  See http://docs.jboss.org/hibernate/orm/4.2/manual/en-US/html_single/#types-value-basic